README.asciidoc

```diff
@@ -0,0 +1,17 @@
+Maven Base Docker Image
+-----------------------
+
+Building
+~~~~~~~~
+
+`docker build -t base-maven .`
+
+Example Usage
+~~~~~
+
+* `cd [workDir]`
+
+* `docker run --privileged=true -v {backtick}pwd{backtick}:/opt/jboss/myproject base-maven mvn clean install -f /opt/jboss/myproject`
```

Comment:
Needed this to be able to let the java/mvn process have access to the mounted filesystem; in this example we have the Docker container building sources contained on the host FS.  Better ideas to enable this without selinux policy violations?